stack/stack.go

```diff
@@ -1,39 +1,91 @@
 package main
 
+//Imported Libraries
 import (
-	"errors"
 	"fmt"
 )
 
-// Stack implements stack DS
-type Stack struct {
-	list []int
+//Constants
+const x  = 10
+
+//Implementation of the stack structure
+type stack struct {
+	items [x] string
+}
+//pop from the stack
+func (a *stack) push (value string){
+	var found bool = false
+	for index,valuea := range a.items{
+		if valuea == "" {
+			a.items[index]=value
+			found=true
+			break
+		}
+	}
+	if found == false{
```

Comment:
Formatting can be improved.

*Origin: GofmtBear, Section: `gofmt`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpgx2e53q9/stack/stack.go
+++ b/tmp/tmpgx2e53q9/stack/stack.go
@@ -22,7 +22,7 @@
 			break
 		}
 	}
-	if found == false{
+	if found == false {
 		fmt.Printf("Stack is full, Try on PUSH something outside\n")
 	}
 }
```